completions/bash/podman

```diff
@@ -818,7 +815,6 @@ _podman_import() {
      "
     local boolean_options="
 	--help
-	-h
```

Comment:
I believe the -h will still work for --help for all commands except for podman run/create.  So these should not be removed?

Reply:
good point, the -h will still work for the commands other than run/create, I missed that here....